nemo/collections/nlp/models/intent_slot_classification/intent_slot_classification_model.py

```diff
@@ -108,8 +107,16 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             mode='micro',
         )
 
-        # Optimizer setup needs to happen after all model weights are ready
-        self.setup_optimization(cfg.optim)
+    def update_data_dir(self, data_dir: str) -> None:
```

Comment:
Does this solve the remaining tlt issues?

Reply:
It solve some of the issues that I can potentially change data directory for fine tuning and evaluation. But for training I still need data set to sit inside of the model config file for the model initialization.